targets/posix/nanoCLR/targetHAL_Time.cpp

```
@@ -0,0 +1,129 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include <chrono>
+#include <cstdint>
+#include <cstdio>
+#include <thread>
+#include <time.h>
```

Clang (14.0.6)

[warning] inclusion of deprecated C++ header 'time.h'; consider using 'ctime' instead

(modernize-deprecated-headers)